sql/sql_alter.cc

```diff
@@ -346,6 +347,43 @@ bool Sql_cmd_alter_table::execute(THD *thd) {
   if (check_grant(thd, priv_needed, first_table, false, UINT_MAX, false))
     return true; /* purecov: inspected */
 
+#ifdef WITH_WSREP
+  /* Check if foreign keys are accessible.
+  1. Transaction is replicated first, then is done locally.
```

Comment:
Just for the clarity, Does it mean that it is first replicated and then its execution is done locally? If yes, please update the comment.
```suggestion
  1. Transaction is replicated first, then is executed locally.
```

Reply:
fixed